src/components/EditEvent/EditEvent.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEvents } from "../../hooks/useEvents";
+
+import "@/components/AddEvent/AddEvent.css";
+import { useParams } from "react-router-dom";
+import { EventFormHelper } from "../EventFormHelper/EventFormHelper";
+
+export function EditEvent() {
+  const { id } = useParams();
+  const { events } = useEvents();
+
+  if (!id) {
+    return <h1>Please provide id</h1>;
+  }
+
+  const initFormData = events.find(({ id: eventId }) => eventId === +id);
```

Comment:
what is `+` here??

Reply:
to convert the `id` that is a `string` we got from route parameters to a `number`

Reply:
make sure you add comments for such code